core/service/src/lib.rs

```diff
@@ -185,10 +185,6 @@ impl<Components: components::Components> Service<Components> {
 		// This is meant to be for testing only
 		// FIXME #1063 remove this
 		if let Some(keystore) = keystore.as_mut() {
-			for seed in &config.keys {
```

Comment:
I removed this without a replacement, since FIXME #1063 suggests removing it. I can add it back to the store if it's still required for something...